### ISSUE TITLE
Results shown outside price range selected

### ISSUE THREAD

**Issue by jordycosta** [iFixit/react-commerce#533]
### Overview:

When selecting one of the price range facet options, there are some results that are displayed to users that are out of bounds from what they chose. These results are included because the product has either a Fix Kit or a Part Only option that is included in that price range.

![price_range](https://user-images.githubusercontent.com/95656772/181648006-5654db60-b88a-4bef-9b8d-f6e236d3aee3.jpg)

![part-only](https://user-images.githubusercontent.com/95656772/181648247-3a332569-8d8f-4539-a32b-08d06213beb5.jpg) ![fix-kit](https://user-images.githubusercontent.com/95656772/181648254-096fb737-e8f2-45a1-adfa-1fa554e7bb5b.jpg)

---

**Comment by masonmcelvain** [comment]
Related: https://github.com/iFixit/ifixit/pull/43862

**Comment by sterlinghirsh** [comment]
@mmarcon91 Can we experiment with putting a price range on the product card again?

**Comment by dhmacs** [comment]
@sterlinghirsh Just to double check, is it ok that we're using `price_float` instead of for example `group_min_price` to display the product price?

**Comment by mmarcon91** [comment]
> @mmarcon91 Can we experiment with putting a price range on the product card again?

I do some research to understand how to manage different prices. My concern is when we have to show discounts, it would be a bit confusing to have two ranges

**Comment by sterlinghirsh** [comment]
>  is it ok that we're using price_float instead of for example group_min_price to display the product price?

I think it's okay, though it's the price of the default variant, which isn't necessarily the lowest. It's confusing when it shows one price on search but another price after clicking the link.

**Comment by mmarcon91** [comment]
@sterlinghirsh I did a bit of research to understand how other sites handle this case and actually their approach is as you suggested to have min and max price.
I think it is the most effective solution.

I took the opportunity to create a new component of the product card to keep as a reference. prices have been resized to avoid taking up too much space.
https://www.figma.com/file/sfj6r7Bn6eREMvuqo0f7Du/iFixit---Components?node-id=308%3A17651

Here is a preview of the collection with the updated product cards.
https://www.figma.com/file/jRG4EyQXRDMGfyLqut08KP/iFixit---Working-files?node-id=5677%3A63004
